packages/vue-client/src/components/GameCreation/BadukConfigForm.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import type { BadukConfig } from "@ogfcommunity/variants-shared";
+
+const props = defineProps<{ initialConfig: BadukConfig }>();
+const config = { ...props.initialConfig } as BadukConfig;
+
+const emit = defineEmits<{
+  (e: "configChanged", config: BadukConfig): void;
+}>();
+
+function emitConfigChange() {
+  emit("configChanged", config);
+}
+</script>
+
+<template>
+  <div class="column">
+    <label>Width</label>
+    <input
+      type="number"
+      min="1"
+      v-model="config.width"
+      @change="emitConfigChange"
+      style="width: fit-content"
+    />
+    <label>Height</label>
+    <input
+      type="number"
+      min="1"
+      v-model="config.height"
+      @change="emitConfigChange"
+      style="width: fit-content"
+    />
+    <label>Komi</label>
+    <input
+      type="number"
+      step="0.5"
+      v-model="config.komi"
+      @change="emitConfigChange"
+      style="width: fit-content"
+    />
+  </div>
+</template>
```

Comment:
Why not just add a CSS rule `input { width: fit-content }` inside a `<style>` block for the component?  Don't repeat yourself (DRY).

Or `.awesomeClassName input { width: fit-content }` somewhere for all config components?

Reply:
added style block and rule